### PR TITLE
chore: bump webpacker 5.4.3

### PR DESCRIPTION
**Pull request by mc-jones**
This PR updates webpacker to the latest stable version to address security vulnerabilities in 5.1.1.

- `tar` (high severity): https://github.com/artsy/horizon/security/dependabot/yarn.lock/tar/open
- `serialize-javascript` (high severity): https://github.com/artsy/horizon/security/dependabot/yarn.lock/serialize-javascript/open
- `trim-newlines` (high severity): https://github.com/artsy/horizon/security/dependabot/yarn.lock/trim-newlines/open

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -27,6 +27,18 @@ module.exports = function (api) {
...
+        {
+          loose: true,
+        },
+      ],
+      [
+        "@babel/plugin-proposal-private-property-in-object",
+        {
+          loose: true,
+        },
+      ],
```
Comment (mc-jones) [inline babel.config.js:41]: `@babel/plugin-proposal-class-properties`, `@babel/plugin-proposal-private-methods` and `@babel/plugin-proposal-private-property-in-object` all need to have the same option for `loose`, per babel docs [here](babel/plugin-proposal-private-methods#loose) and [here](https://babeljs.io/docs/en/babel-plugin-proposal-private-property-in-object#loose).

File: Gemfile.lock [hunk 2]
```diff
@@ -311,7 +311,7 @@ GEM
       rails (>= 3.2.16)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.4.2)
+    zeitwerk (2.5.1)
```
Comment (joeyAghion) [inline Gemfile.lock:314]: Are these upgrades related at all?

**Review by joeyAghion — APPROVED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (mc-jones) [reply Gemfile.lock:314]: They should be `webpacker` dependency updates as they were updated as a result of `bundle update webpacker`. We had updated the Gemfile directly when bumping just the Ruby side of webpacker previously, see [PR](https://github.com/artsy/horizon/commit/92db616ffb7f7f90686df18327a7df24b726e37c), which is why we aren't seeing a direct `webpacker` bump in this PR, but just its dependencies.

For this update, I followed the full upgrade instructions from `webpacker` [here](https://github.com/rails/webpacker#upgrading).
